### PR TITLE
DYN-7279 Add title and name to Agreements for Data Collection window

### PR DESCRIPTION
**Pull request by avidit**
### Purpose

Add title to Agreements for Data Collection window

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

N/A

### Reviewers

(FILL ME IN) Reviewer 1 (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCoreWpf/UI/Prompts/UsageReportingAgreementPrompt.xaml [hunk 1]
```diff
@@ -4,6 +4,8 @@
         xmlns:ui="clr-namespace:Dynamo.UI"
         xmlns:p="clr-namespace:Dynamo.Wpf.Properties;assembly=DynamoCoreWpf"
         xmlns:views="clr-namespace:Dynamo.UI.Views"
+        Title="Agreements for Data Collection"
+        Name="AgreementsForDataCollection"
```
Comment (zeusongit) [inline src/DynamoCoreWpf/UI/Prompts/UsageReportingAgreementPrompt.xaml:8]: Can we use a resource string instead...for localization?

Comment (avidit) [reply src/DynamoCoreWpf/UI/Prompts/UsageReportingAgreementPrompt.xaml:8]: I need the `Name` for this window to use as unique id in automated tests. This window doesn't have one. I think we can localize `Title`, but I have never done this before, can you please give me an example?

Comment (QilongTang) [reply src/DynamoCoreWpf/UI/Prompts/UsageReportingAgreementPrompt.xaml:8]: Here is an example: https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCoreWpf/Views/Debug/DebugModesWindow.xaml#L5

**Review by zeusongit — APPROVED** [review]

**Comment by zeusongit** [comment]
One test failed:
[DynamoCoreWpfTests.JSONSerializationTests.SerializationTest](https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/16629/testReport/junit/DynamoCoreWpfTests/JSONSerializationTests/SerializationTest__D___jenkins__workspace__Dynamo__DynamoSelfServe__pullRequestValidation__Dynamo__test__core__CustomNodes__TestDefaultValue_dyn__/)
Related?

**Comment by QilongTang** [comment]
Rerunning the PR check returns a different set of failures at https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/16646/

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
Why would Jira ticket check fail as well? PTAL

**Comment by avidit** [comment]
> Why would Jira ticket check fail as well? PTAL

It is passing now.

**Pull request merged**